packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -5,12 +5,10 @@ import * as fs from 'fs';
 import * as minimatch from 'minimatch';
 import * as path from 'path';
 
-export interface DockerImageAssetProps extends assets.FingerprintOptions {
-  /**
-   * The directory where the Dockerfile is stored
-   */
-  readonly directory: string;
-
+/**
+ * Options for DockerImageAsset
```

Comment:
Do you think we can be more descriptive as to when users should use DockerImageAssetOptions vs DockerImageAssetProps? (we've had users who have been confused with ContainerDefinitionOptions vs ContainerDefinitionProps in the past)

Reply:
Not sure there is something super smart to write here... Maybe something like "includes all the optional configuration" (without the mandatory one)...

Reply:
I'll update in a subsequent commit. Let me know what you think.